test/configCases/container/eager-async/index.js

```diff
@@ -0,0 +1,9 @@
+it("should load shared module eagerly", async () => {
+	const reactValue = require("react");
```

Comment:
notice that no import() is used in the example, and the shared react is not eager